src/canmatrix/canmatrix.py

```diff
@@ -853,8 +951,8 @@ def saveConvertStrToInt(inStr):
         elif definition[0:3] == 'HEX':
             self.type = 'HEX'
             min, max = definition[4:].split(' ', 2)
-            self.min = saveConvertStrToInt(min)
-            self.max = saveConvertStrToInt(max)
+            self.min = safeConvertStrToInt(min)  # the same for INT and HEX?
```

Comment:
I agree, that doesn't sound right.  But, I don't know offhand what triggers this code.